excel-importer/src/main/java/uk/gov/hmcts/ccd/definition/store/excel/parser/ComplexFieldTypeParser.java

```diff
@@ -118,6 +127,8 @@ private ComplexFieldEntity parseComplexField(DefinitionDataItem definitionDataIt
         complexField.setOrder(order);
         complexField.setShowCondition(parseShowCondition(definitionDataItem.getString(ColumnName.FIELD_SHOW_CONDITION)));
         complexField.setDisplayContextParameter(definitionDataItem.getString(ColumnName.DISPLAY_CONTEXT_PARAMETER));
+        complexField.setSearchable(definitionDataItem.getBooleanOrDefault(ColumnName.SEARCHABLE, true));
```

Comment:
this looks like its was added by a merge from develop - not part of RDM-9024

Reply:
Thanks @RebeccaBaker, that line is commented out now